less/index.less

```diff
@@ -57,3 +57,145 @@ html, body {
     height: 100%;
 }
 
+@link-color:        #428bca; // sea blue
+@link-color-hover:  darken(@link-color, 10%);
+@font-color: darkgray; 
+@font-type: Arial; 
+
+/* Your code starts here! */
+.container {
+	width: 900px; 
+	margin: auto;
+
+	@media (max-width: 500px) {
+		width: 100%;
+		
+	} 	
+}
+  
+header nav {
+ display: flex;
+  margin-top: 35px;
+ 
+  @media (max-width: 500px) {
+	display: flex; 
+	flex-direction: column; 
+ 
+}
+}
+
+.logo {
+	margin-left:650px;
+	display: flex;
+	margin-top: -50px;
+
+	
+}
+
+header nav a {
+	margin: 20px; 
+	color: @link-color;
+}
+
+
+
+.sub img{
+display: flex;
+margin-left: 30%;
+    margin-bottom: 50px;
+	margin-top: 50px;
+}
```

Comment:
So, setting a percentage margin could end up looking funky on certain screen widths. One way to better add margins here and _also_ get a centered image is to use the shorthand property:

`margin: 50px auto;`

This sets 50px margin on the top and bottom, and auto on the left and right which will take up the remaining space between the two margins.